### DISCUSSION TITLE
Any pre-built windows version?

### DISCUSSION THREAD

**Discussion by peepshow-21** [berry-lang/berry#228] [General]
Hi, as per subject.

**Comment by s-hadinger** [comment]
No sorry.

**Comment by skiars** [comment]
You can try using MinGW, I do it often.

**Comment by rj10rd** [comment]
How use berry in windows

**Reply by s-hadinger** [reply]
Already answered, use MinGW

**Reply by s-hadinger** [reply]
There is a pre-built Win version here: https://github.com/arendst/Tasmota/blob/development/lib/libesp32/berry/berry.exe

It is built with w64devkit (which includes MinGW)

**Reply by rj10rd** [reply]
thanks for the answer,I am a noob can you please tell me how to use berry in vs code

**Reply by rj10rd** [reply]
if possible can you please make detailed tutorial to learn berry on Youtube
I am learning my first language, thnx

**Reply by s-hadinger** [reply]
Sorry to ask but why did you choose berry to learn programming. Why not start with Python. 

There is no plan for a YouTube video
